src/transforms/parse.js

```diff
@@ -19,8 +19,9 @@ module.exports = function(value, outputPath) {
 			const tocLabel = document.createElement("span");
 			const tocUl = document.createElement("ul");
 			tocNav.setAttribute("aria-labelledby", "toc-nav-label");
```

Comment:
I think you can directly use `aria-label` to get rid of the necessity of using an extra `<span>` element for holding the label text.